docs/docs/references/users/README.md

```diff
@@ -1,5 +1,91 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information.
+
+A user profile consists of a few sections:
```

Comment:
Maybe `consist of the following types of data` be better, sections more like a well-defined structure, which technically we are not following.

Reply:
updated

```
A user profile consists of a few categories:
```